tests/letstest/scripts/test_letsencrypt_auto_certonly_standalone.sh

```diff
@@ -14,7 +14,7 @@ letsencrypt-auto certonly --no-self-upgrade -v --standalone --debug \
                    --text --agree-dev-preview --agree-tos \
                    --renew-by-default --redirect \
                    --register-unsafely-without-email \
-                   --domain $PUBLIC_HOSTNAME --server $BOULDER_URL
```

Comment:
Oops, I just noticed these changes were pulled in by accident because of the shell linter I use. They're not necessary, but I feel like leaving them in should be OK?